apps/site/src/app/(main)/portal/@applicant/ApplicantPortal.tsx

```diff
@@ -28,14 +28,14 @@ async function Portal() {
 	const submittedWaiver =
 		status === PortalStatus.waived || status === PortalStatus.confirmed;
 
-	const isAccepted = status === PortalStatus.accepted || submittedWaiver;
+	const isAccepted = status === PortalStatus.accepted;
 
 	return (
 		<div className="bg-white text-black max-w-4xl rounded-2xl p-6 flex flex-col mb-24 w-full">
 			<h2 className="text-4xl font-semibold">Status</h2>
 			<VerticalTimeline status={status as PortalStatus} />
 			<Message status={status as PortalStatus} />
-			{isAccepted && <SignWaiver status={status as PortalStatus} />}
+			{isAccepted && !submittedWaiver && <SignWaiver />}
```

Comment:
Thought: all of the states are mutually exclusive, so we shouldn't need the second conditional. Perhaps consider renaming `isAccepted` to `needsToSignWaiver`?